### ISSUE TITLE
[Feature request] object-key syntax

### ISSUE THREAD

**Issue by boris-petrov** [pzuraq/ember-simple-set-helper#4]
[In the documentation about the differences with ember-set-helper](https://github.com/pzuraq/ember-simple-set-helper#differences-from-ember-set-helper) it's written that only the placeholder syntax is a difference. But, `ember-simple-set-helper` also doesn't support the [object-key syntax](https://github.com/pzuraq/ember-set-helper#usage). So either the documentation should be updated, or the feature implemented. I would vote for the latter as I'm using it and I need it. :)

**Comment by viniciussbs** [comment]
@boris-petrov I don't know if this works, but have you tried something like this?

```hbs
<button {{on "click" (set (get this "greeting") "Hello!")}}>
  English
</button>
```

**Comment by boris-petrov** [comment]
@viniciussbs - I haven't tried it but what you suggest most likely won't work. It is like writing in the JavaScript code `this.set(this.get('greeting'), 'Hello!')` - which doesn't work.

**Comment by viniciussbs** [comment]
Yeah, it makes sense. About the issue, though, I think it can be closed since `ember-simple-set-helper` does support object-key syntax and [it's now documented](https://github.com/pzuraq/ember-set-helper#usage)?

**Comment by boris-petrov** [comment]
You've linked to `ember-set-helper`'s documentation, this here is `ember-simple-set-helper`. :smile:

**Comment by viniciussbs** [comment]
Your're right again. 🤦🏽‍♂️ So, yeah, it should be documented, because it works? https://github.com/pzuraq/ember-simple-set-helper/blob/master/lib/simple-set-transform.js

**Comment by boris-petrov** [comment]
When I opened the issue, it didn't. I haven't tried since. I'm not sure I can use `ember-simple-set-helper` any more because I'm using the placeholder syntax but if you confirm the object-key syntax works, I'll close the issue.
